packages/@okta/vuepress-site/docs/guides/sign-into-web-app/main/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Sign users in to your web application
+excerpt: Learn how to add authentication to your web applications and sign users in using Okta's APIs and libraries.
+layout: Guides
+---
+
+This guide shows you how to use Okta as the user store for your server-side web application and sign users in.
+
+> **Note**: If you are building a single-page (browser) app, see [Sign users in to your single-page application](/docs/guides/sign-into-spa/) instead. Or, if you are building a server that returns API responses (but not HTML), see [Protect your API endpoints](/docs/guides/protect-your-api/).
+
+---
+
+**Learning outcomes**
+
+* Create an Okta app integration to represent your app in your org.
+* Add dependencies and configure your app ready to use Okta for authentication.
+* Sign a user in to your app. 
+
+**What you need**
+
+* An Okta Developer Edition organization. Don't have one? [Create one for free](https://developer.okta.com/signup).
+* Knowledge of building server-side Web applications.
+* A project or application that you want to add authentication to.
+
+**Sample code**
+
+* ASP.NET — see our [ASP.NET MVC example](https://github.com/okta/samples-aspnet/tree/master/okta-hosted-login) for a working sample app, or check out [ASP.NET MVC quickstart](https://docs.microsoft.com/en-us/aspnet/mvc/overview/getting-started/introduction/getting-started) if you want to build your own to work through this guide with.
```

Comment:
Our style is to use em-dashes here, not hypens; applies to all bullets

Reply:
Those were literal em-dash characters, created using alt + shift + - on the mac keyboard, but I appreciate it is easier to reason with if you use em-dash HTML entity refs. I've changed them, and will use them in future.

Reply:
those were literal em-dash characters (created using Alt + Shift + - on mac keyboard), but I appreciate HTML entities are probably easier to reason with, so I've changed to those